<launch>

	<node pkg="navfn" name="planner" type="navfn_node"\>
		


